config/initializers/will_paginate.rb

```diff
@@ -23,8 +23,8 @@ def forward
       end
 
       def pages
-        current_page_input = tag(:input, '', class: 'pagination-pf-page', type: 'text', value: current_page, id: 'pagination1-page')
-        current_page_label = tag(:label, _('Current Page'), class: 'sr-only', for: 'pagination1-page')
+        current_page_input = tag(:input, '', class: 'pagination-pf-page', type: 'text', value: current_page, id: 'cur-page-num')
```

Comment:
The id here is `cur-page-num` but below you look for `#cur_page_num` so it fails to find the right element. The tests also use `cur_page_num` which is why they pass.